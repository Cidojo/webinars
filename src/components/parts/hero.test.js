import * as React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import {shallow, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import {Hero} from './Hero.jsx';

configure({adapter: new Adapter()});

const props = {
  onAddNewClick: () => {}
};

it(`should render Hero with add button component without errors`, () => {
  const tree = shallow(<Hero {...props}/>);
  expect(toJson(tree)).toMatchSnapshot();
});

it(`should render Hero without add button component without errors`, () => {
  const tree = shallow(<Hero {...props} withButton={false} />);
  expect(toJson(tree)).toMatchSnapshot();
});
