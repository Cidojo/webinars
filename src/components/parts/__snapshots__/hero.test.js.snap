// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Hero with add button component without errors 1`] = `
<section
  className="hero"
>
  <div
    className="hero__inner"
  >
    <section
      className="hero__description"
    >
      <h1
        className="hero__title"
      >
        Webinars
      </h1>
      <small
        className="hero__text"
      >
        Here you can register and take part in educational webinars conducted by the best digital marketing experts.
      </small>
      <button
        className="button-primary"
        onClick={[Function]}
      >
        Add new
      </button>
    </section>
  </div>
</section>
`;

exports[`should render Hero without add button component without errors 1`] = `
<section
  className="hero"
>
  <div
    className="hero__inner"
  >
    <section
      className="hero__description"
    >
      <h1
        className="hero__title"
      >
        Webinars
      </h1>
      <small
        className="hero__text"
      >
        Here you can register and take part in educational webinars conducted by the best digital marketing experts.
      </small>
    </section>
  </div>
</section>
`;
